### DISCUSSION TITLE
Which active learning model should I choose in ASReview LAB?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#555] [Q&A] [answered]
When setting up my project, I can choose between several classifiers for example. But how do I know which classifier to use? And related, which feature extraction method and query strategy?

**Comment by terrymyc** [comment, marked as answer]
In ASReview LAB, you need to choose which model you want to use to predict relevancy of your documents. Currently, we always advise to use the Naive Bayes classifier, TF-IDF feature extraction technique, and maximum query strategy since they perform very well and need little computation time. We have performed several simulation studies to evaluate performance of different models on several datasets. See [simulation results](https://asreview.readthedocs.io/en/latest/guides/simulation_study_results.html#simulation-results). We do not advise specific models for specific jobs because we've not found enough evidence (yet) to make such recommendations.
